modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLViewEntriesDisplayContext.java

```diff
@@ -270,8 +271,10 @@ public SearchContainer<RepositoryEntry> getSearchContainer()
 	}
 
 	public String getThumbnailSrc(FileVersion fileVersion) throws Exception {
-		return DLURLHelperUtil.getThumbnailSrc(
-			fileVersion.getFileEntry(), fileVersion, _themeDisplay);
+		return _addDoAsUserIdParameter(
```

Comment:
Why are you passing the ThemeDisplay as parameter if it is an attribute?

Reply:
Changed, in both places